src/main/java/teammates/common/datatransfer/attributes/AccountAttributes.java

```diff
@@ -26,49 +27,95 @@
     public Date createdAt;
     public StudentProfileAttributes studentProfile;
 
-    public AccountAttributes(Account a) {
```

Comment:
Any benefit of removing the constructor? I think we should keep it there as this gives us a way to convert `entity` to `datatransfer` object.

Reply:
Makes sense. Do you also want me to revert the 2-3 places where I used builder instead of this constructor?

Reply:
Go ahead :)

Reply:
Will it make more sense to use `valueOf()` instead of the constructor.
Similar method is used in other *Attributes classes

Reply:
Can try to make it consistent :)